src/containers/SearchPage/searchHelpers.js

```diff
@@ -235,3 +235,162 @@ export const getResultMetadata = search => ({
   totalCountSubjectMaterial: search.totalCountSubjectMaterial || 0,
   totalCountTasks: search.totalCountTasks || 0,
 });
+
+const searchTypeFilterOptions = {
+  subject: [],
+  'topic-article': [],
+  'learning-path': [],
+  'subject-material': [
+    {
+      name: 'Veiledning',
+      id: 'urn:resourcetype:guidance',
+    },
+    {
+      name: 'Forelesning og presentasjon',
+      id: 'urn:resourcetype:lectureAndPresentation',
+    },
+    {
+      name: 'Fagartikkel',
+      id: 'urn:resourcetype:academicArticle',
+    },
+  ],
+  'tasks-and-activities': [
+    {
+      name: 'Oppgave',
+      id: 'urn:resourcetype:task',
+    },
+    {
+      name: 'Øvelse',
+      id: 'urn:resourcetype:exercise',
+    },
+  ],
+  'assessment-resources': [],
+  'external-learning-resources': [],
+  'source-material': [],
+};
+
+export const searchSubjectTypeOptions = t => [
+  {
+    title: t('contentTypes.all'),
+    value: 'ALL',
+  },
+  {
+    title: t('contentTypes.topic'),
+    value: 'topic-article',
+  },
+  {
+    title: t('contentTypes.subject-material'),
+    value: 'subject-material',
+  },
+  {
+    title: t('contentTypes.learning-path'),
+    value: 'learning-path',
+  },
+  {
+    title: t('contentTypes.tasks-and-activities'),
+    value: 'tasks-and-activities',
+  },
+  {
+    title: t('contentTypes.assessment-resources'),
+    value: 'assessment-resources',
+  },
+  {
+    title: t('contentTypes.external-learning-resources'),
+    value: 'external-learning-resources',
+  },
+  {
+    title: t('contentTypes.source-material'),
+    value: 'source-material',
+  },
+];
+
+export const filterTypeOptions = (searchGroups, t) =>
+  searchSubjectTypeOptions(t).filter(
+    option =>
+      searchGroups.find(group => group.type === option.value)?.items?.length ||
+      option.value === 'ALL',
+  );
+
+export const mapResourcesToItems = resources =>
+  resources.map(resource => ({
+    id: resource.id,
+    title: resource.name,
+    ingress: resource.ingress,
+    breadcrumb: resource.breadcrumb,
+    url: `${toSubjects()}${resource.path}`,
+    ...(resource.img?.url && {
+      img: {
+        url: `${resource.img.url}?focalX=50&focalY=50&ratio=1.75`,
```

Comment:
Bruk av focal sammen med ration gjør ingenting med bildet og laster det ned i full størrelse. Veit det står i swagger-dok men det er bedre å spesifisere width=250 heller. For bildet https://api.test.ndla.no/image-api/raw/id/42683 fører disse parameterene til et bilde på 2.3MB mens med width=250 blir et bilde på 82K.

Reply:
Kanskje vi skal bruke src-set som vi gjør for framvisning av bilder via article-converter. Vi bør uansett gjøre noko med dette.

Reply:
Så ikke at det fantes parametre for det i image-api. Skal fikse det